Running `make pdk` causes an error if the PDK is already installed

### Description
Upon cloning the OpenLane repo, one of the steps that must be done is `make pdk`.

This Makefile recipe clones the correct repository and installs a PDK locally.

Attempting to run this Makefile recipe again, such as for the purposes of installing a new PDK or updating, leads to an error.
This is due to the old PDK installation now existing, blocking the Makefile recipe [here](https://github.com/The-OpenROAD-Project/OpenLane/blob/master/dependencies/pdk.mk#L39).

### Reproduction Material
```
tdene@orion:/data/OpenLane$ make pdk
git clone https://github.com/google/skywater-pdk /data/OpenLane/pdks/skywater-pdk
fatal: destination path '/data/OpenLane/pdks/skywater-pdk' already exists and is not an empty directory.
make: *** [dependencies/pdk.mk:40: /data/OpenLane/pdks/skywater-pdk/LICENSE] Error 128
```

### Suggested fix

See pull request #922 (to be edited in within the next 5 minutes, once GitHub assigns a number

--- a/dependencies/pdk.mk
+++ b/dependencies/pdk.mk
@@ -36,7 +36,7 @@ pdk: skywater-pdk skywater-library open_pdks build-pdk gen-sources
 $(PDK_ROOT):
 	mkdir -p $(PDK_ROOT)
 
-$(PDK_ROOT)/skywater-pdk/LICENSE: $(PDK_ROOT)
+$(PDK_ROOT)/skywater-pdk/LICENSE: | $(PDK_ROOT)
 	git clone $(shell $(PYTHON_BIN) ./dependencies/tool.py sky130 -f repo) $(PDK_ROOT)/skywater-pdk
 
 .PHONY: skywater-pdk

